src/base/datetime/calendar/header/dropdown/month/index.ts

```diff
@@ -0,0 +1,229 @@
+    event.preventDefault();
+  }
+
+  private _handleMouseDownDropdownToggle(event: MouseEvent) {
+    event.preventDefault();
+  }
+
+  private _handleKeyDownMonthToggle(event: KeyboardEvent) {
+    if (!this._menuVisible) {
+      this._menuEl.highlightFirstItem();
+      return;
+    }
+    switch (event.key) {
+      case "Up":
+      case "ArrowUp": {
+        event.preventDefault();
+        this._menuEl.highlightPrevItem();
+        this._setActiveDescendant(
+          this._toggleEl,
+          this._menuEl.getHighlightItemId()
+        );
+        break;
+      }
+      case "Down":
+      case "ArrowDown": {
+        event.preventDefault();
+        this._menuEl.highlightNextItem();
+        this._setActiveDescendant(
+          this._toggleEl,
+          this._menuEl.getHighlightItemId()
+        );
+        break;
+      }
+      case "Home":
+        event.preventDefault();
+        this._menuEl.highlightFirstItem();
+        this._setActiveDescendant(
+          this._toggleEl,
+          this._menuEl.getHighlightItemId()
+        );
+        break;
+      case "End":
+        event.preventDefault();
+        this._menuEl.highlightLastItem();
+        this._setActiveDescendant(
+          this._toggleEl,
+          this._menuEl.getHighlightItemId()
+        );
+        break;
+      case "Enter": {
+        event.preventDefault();
+        const highlightValue = this._menuEl.getHighlightValue();
+        if (highlightValue) {
+          this.month = Number(highlightValue);
+          const detail: CustomEventDetail = { value: `${this.month}` };
+          dispatchCustomEvent(this, "kuc:month-dropdown-change", detail);
+        }
+        this._menuVisible = false;
+        break;
+      }
+    }
+  }
+
+  private _setActiveDescendant(
+    _buttonEl: HTMLButtonElement,
+    value?: string | null
+  ) {
+    if (value && _buttonEl !== null) {
+      _buttonEl.setAttribute("aria-activedescendant", value);
+    }
+  }
+
+  private _handleBlurDropdownMonthToggle(event: Event) {
+    this._menuVisible = false;
+  }
+
+  private _handleChangeMenu(event: CustomEvent) {
+    event.preventDefault();
+    event.stopPropagation();
+    this.month = Number(event.detail.value);
+    this._menuVisible = false;
+    const detail: CustomEventDetail = { value: `${this.month}` };
+    dispatchCustomEvent(this, "kuc:month-dropdown-change", detail);
+  }
+
+  private _openMenu() {
+    this._toggleEl.focus();
+    this._menuVisible = true;
+    this._menuEl.highlightSelectedItem();
+  }
+
+  private _closeMenu() {
+    this._menuVisible = false;
+    this._removeActiveDescendant(this._toggleEl);
+  }
+
+  private _removeActiveDescendant(_buttonEl: HTMLButtonElement) {
+    _buttonEl.removeAttribute("aria-activedescendant");
+  }
+
+  private _getMenuItems() {
+    return this._locale.MONTHS_SELECT.map((month: string, index: number) => {
+      const item: Item = {
+        value: `${index + 1}`,
+        label: `${month}`,
```

Comment:
Delete the unnecessary comma.

```suggestion
        label: `${month}`
```